components/button.css

```diff
@@ -0,0 +1,128 @@
+.btn{
+    padding: 0.5rem 1rem;
+    border: none;
+    border-radius: 5px;
+    color: white;
+}
+
+.btn:hover{
+    background-image: linear-gradient(rgba(0, 0, 0, 0.2) 0 0);
+    cursor: pointer;
+}
+
+/* default */
+.btn-primary{
+    background-color: var(--primary-color);
+}
+
+.btn-success{
+    background-color: var(--success-color);
+}
+
+.btn-warning{
+    background-color: var(--warning-color);
+}
+
+.btn-error{
+    background-color: var(--danger-color);
+}
+
+.btn-dark{
+    background-color: var(--dark-color);
+}
+
+
+/* outline */
+.btn-outline-primary{
+    background-color: transparent;
+    color: var(--primary-color);
+    border: 1px solid var(--primary-color);
+}
+
+.btn-outline-success{
+    background-color: transparent;
+    color: var(--success-color);
+    border: 1px solid var(--success-color);
```

Comment:
Use utility class for border width and style or add in base class

Reply:
as you suggested i have made changes to the code